main/webapp/modules/core/scripts/project/browsing-engine.js

```diff
@@ -234,6 +242,22 @@ BrowsingEngine.prototype.removeFacet = function(facet) {
   }
 };
 
+BrowsingEngine.prototype.hideShowBrowsingFacetSplash = function(clicked) {
```

Comment:
No need of Browsing is this function name. It's important for the prefs (category of pref) but not here where it's implied. Every thing here is abour browsing.